### PR TITLE
Mobile portfolio setup 

### PR DESCRIPTION
**Pull request by Dinicharia**
This is my branch and this was created to build the header section of my portfolio.
I am going to add the HTML, CSS, and images for the project.
Make sure there are no linter errors.
This follows the Guthub workflow

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>setup and mobile first</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <nav class="hdr" >
+            <a href="#"><img src="img/MyLogo.png" alt="logo"></a> 
+            <a href="#"> <img src="img/iconmenu.png" alt="iconmenu"></a>
```
[the remark this answers is not in the document]
Comment (Dinicharia) [reply index.html:14]: Hello,
My hamburger icon looks okay. I'm not sure why it looks different on your end.
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>setup and mobile first</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <nav class="hdr" >
+            <a href="#"><img src="img/MyLogo.png" alt="logo"></a>
```
[the remark this answers is not in the document]
Comment (Dinicharia) [reply index.html]: Changed class names to more meaningful names.
Removed the logo from inside the nav tag.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>setup and mobile first</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
```
[the remark this answers is not in the document]
Comment (Dinicharia) [reply index.html:11]: Wrapped only the toolbar in the header tag.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+                    <li> <a href="https://www.linkedin.com/in/daniel-njoroge-3651a2a5/" target="_blank" rel="noopener"><img src="img/Linkedin.png" alt="linkedin" ></a> </li> 
+                    <li> <a href="https://twitter.com/Icharian" target="_blank" rel="noopener"> <img src="img/Twitter.png" alt="twitter"></a></li>    
+                    <li> <a href="https://github.com/Dinicharia" target="_blank" rel="noopener"> <img src="img/github.png" alt="github"></a></li>
+                    <li> <a href="www.angelink.com" target="_blank" rel="noopener"> <img src="img/angela.png" alt="angela"></a></li>
+                    <li> <a href="www.medium.com" target="_blank" rel="noopener"> <img src="img/M.png" alt="m"></a></li>
+                </ul>
+            
+        </div>
+            </div>
+        </div>
```
[the remark this answers is not in the document]
Comment (Dinicharia) [reply index.html]: Changed the wrapper from a div to a section tag.

**Review by ichala — APPROVED** [review]
Hi @Dinicharia,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  

 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

✅ You used the correct Github Flow.
✅ No Linter Errors.
✅ Good Design.


Good job implementing the required changes :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
*************************************
# Optional Suggestions 

- [ ] Great job your website looks awesome! 💯  kindly update your Readme file with the right information to make it look professional


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**